packages/teleport/src/DesktopSession/TdpClientCanvas.tsx

```diff
@@ -0,0 +1,147 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import React, { useEffect, useRef, CSSProperties } from 'react';
+import { RenderData } from 'teleport/lib/tdp/client';
+import useTdpClientCanvas from './useTdpClientCanvas';
+
+export default function TdpClientCanvas(props: Props) {
```

Comment:
could you use this component to a separate directory? (together with its useTdpClientCanvas.tsx)